### PR TITLE
chore: add coverage report

### PR DESCRIPTION
**Pull request by jjaakola-aiven**
# About this change - What it does

References: #xxxxx

# Why this way

### CODE REVIEW

**Comment by jjaakola-aiven** [comment]
I'll remove the last commit after approval and before merging. The commit is for showing that what report is posted as a comment.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -37,11 +37,55 @@ jobs:
...
+    - run: make karapace/version.py
+
+    - name: Post coverage comment
+      id: post_coverage_comment
+      uses: py-cov-action/python-coverage-comment-action@v3
+      with:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        MERGE_COVERAGE_FILES: true
+
+    - name: Store PR coment to be posted
```
Comment (nosahama) [inline .github/workflows/tests.yml:86]: Since you'd remove a commit, there's a tiny typo here: `coment`

**Review by nosahama — APPROVED** [review]
really nice 🚀

File: .github/workflows/tests.yml (hunk 1 above)
Comment (jjaakola-aiven) [reply .github/workflows/tests.yml]: Done.

**Pull request merged**